### DISCUSSION TITLE
SQL Editor missing tabs?

### DISCUSSION THREAD

**Discussion by MDRIDB** [dbeaver/dbeaver#8959] [Q&A] [answered]
- Windows 7Operating system (distribution) and version Win 7 Professional service pack 1
- DBeaver version 6.1.5

#### Connection specification:
- Database name and version

PostgreSQL 8.3.23 (Greenplum Database 5.21.4 build commit:aac177684671c263131dfc442d1808477fcfa767) on x86_64-pc-linux-gnu, compiled by GCC gcc (GCC) 6.2.0, 64-bit compiled on Sep  5 2019 17:59:31|
- Do you use tunnels or proxies (SSH, SOCKS, etc)? 
NO

#### Describe the problem you're observing:

Artist/output/execution log tab is not showing up as shown in

https://github.com/dbeaver/dbeaver/wiki/SQL-Editor

I am able to see only results tab in my SQL Editor

When execute postgresql function like

select test_function;

I am not able to see the Function's output messages, it show function succeeded in Result tab 

Thanks for guidance.

**Comment by MDRIDB** [comment]
I opened this issue few weeks ago, but I did not hear.

Is this a known issues?

How do I get postgresql function's **"RAISE NOTICE"** messages i Dbeaver? 

How do I get additional output windows to get messages?

Thanks for your guidance.

**Comment by uslss** [comment, marked as answer]
Hi @MDRIDB ,
sorry for the late response.
You could open Output View bi clicking button in SQL editor or using Ctrl+Shift+O
![image](https://user-images.githubusercontent.com/31996417/88171259-ae670780-cc27-11ea-90c5-83d820c80d9b.png)
